git/answers/lui.md

```diff
@@ -0,0 +1,101 @@
+QUESTIONS
+==============================================================
+
+
+What is git? Why do we need it?
+--------------------------------------------------------------
+
+Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordinating work among programmers who are collaboratively developing source code during software development. 
+
+
+What are the top 10 commands? What do they do?
+--------------------------------------------------------------
+
+### Getting started
+
+```
+git clone https://github.com/fractal-bootcamp/precourse-materials
+```
+Downloads existing version of a code project from a remote repository (e.g. Github) and saves it to your computer.
+
+```
+git branch
+```
+See what branches for the repo you have locally.
+
+```
+git branch lui/git
```

Comment:
`git branch -b lui/git`

-b modifies the `branch` command, and creates a new branch

Reply:
I don't think that -b modifier is usable here. There's no sign of it in man git-branch and when I tried it I got:

error: unknown switch `b'

the original command without the -b has been working fine